community/shell/src/main/java/org/neo4j/shell/impl/JLineConsole.java

```diff
@@ -40,7 +40,7 @@ static JLineConsole newConsoleOrNullIfNotFound( ShellClient client )
 				Class.forName( "jline.console.ConsoleReader" ).newInstance();
 			consoleReader.getClass().getMethod( "setBellEnabled",
 				Boolean.TYPE ).invoke( consoleReader, false );
-			consoleReader.getClass().getMethod( "setBellEnabled", Boolean.TYPE ).invoke(
+			consoleReader.getClass().getMethod( "setExpandEvents", Boolean.TYPE ).invoke(
```

Comment:
With this setup `bellEnabled` would not be set to `false`. Is this ok? Maybe setting `expandEvents` to `false` should be a new separate statement?